### PR TITLE
Complement unstable parts of the selenium tests with info about known issues

### PR DESCRIPTION
**Pull request by musienko-maxim**
### What does this PR do?
* We have the selenium tests with unstable parts with dependent issues. In this PR has been aggregated information with typical fails in last 10 jobs on CI and added info  about issues in assertions.

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/7888
@tolusha @vparfonov @dmytro-ndp

### CODE REVIEW

**Comment by musienko-maxim** [comment]
ci-build

**Review by vparfonov — APPROVED** [review]

**Review by tolusha — APPROVED** [review]

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/factory/DirectUrlFactoryWithRootFolder.java [hunk 1]
```diff
@@ -84,7 +84,12 @@ public void factoryWithDirectUrlWithRootFolder() throws Exception {
             "tslint.json",
             "typings.json",
             "wallaby.js");
-    testFactoryWithRootFolder.authenticateAndOpen();
+    try {
+      testFactoryWithRootFolder.authenticateAndOpen();
+    } catch (TimeoutException ex) {
+      // remove try-catch block after issue has been resolved
+      fail("Known issue https://github.com/eclipse/che/issues/7555");
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/factory/DirectUrlFactoryWithRootFolder.java]: It's better to pass **ex** variable to `fail()` method to have additional info about the catching exception.

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/factory/DirectUrlFactoryWithSpecificBranch.java [hunk 2]
```diff
@@ -64,7 +66,13 @@ public void tearDown() throws Exception {
 
   @Test
   public void factoryWithDirectUrlWithSpecificBranch() throws Exception {
-    testFactoryWithSpecificBranch.authenticateAndOpen();
+    try {
+
+      testFactoryWithSpecificBranch.authenticateAndOpen();
+    } catch (TimeoutException ex) {
+      // remove try-catch block after issue has been resolved
+      fail("Known issue https://github.com/eclipse/che/issues/7555");
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/factory/DirectUrlFactoryWithSpecificBranch.java]: It's better to pass **ex** variable to `fail()` method to have additional info about the catching exception.

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/testrunner/JavaTestPluginTestNgTest.java [hunk 3]
```diff
@@ -140,7 +142,13 @@ public void shouldExecuteTestMethodsSuccessfully() throws InterruptedException {
     assertTrue(pluginConsole.getAllNamesOfMethodsMarkedDefinedStatus(PASSED).size() == 1);
     editor.goToCursorPositionVisible(30, 17);
     menu.runCommand(RUN_MENU, TEST, TEST_NG_TEST_DROP_DAWN_ITEM);
-    pluginConsole.waitMethodMarkedAsFailed("shouldFailOfAppAnother");
+    try {
+
+      pluginConsole.waitMethodMarkedAsFailed("shouldFailOfAppAnother");
+    } catch (TimeoutException ex) {
+      // remove try-catch block after issue has been resolved
+      fail("Known issue https://github.com/eclipse/che/issues/7338");
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/testrunner/JavaTestPluginTestNgTest.java]: It's better to pass **ex** variable to `fail()` method to have additional info about the catching exception.

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/testrunner/JavaTestPluginTestNgTest.java [hunk 4]
```diff
@@ -140,7 +142,13 @@ public void shouldExecuteTestMethodsSuccessfully() throws InterruptedException {
     assertTrue(pluginConsole.getAllNamesOfMethodsMarkedDefinedStatus(PASSED).size() == 1);
     editor.goToCursorPositionVisible(30, 17);
     menu.runCommand(RUN_MENU, TEST, TEST_NG_TEST_DROP_DAWN_ITEM);
-    pluginConsole.waitMethodMarkedAsFailed("shouldFailOfAppAnother");
+    try {
+
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/testrunner/JavaTestPluginTestNgTest.java]: redundant empty line

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/factory/DirectUrlFactoryWithSpecificBranch.java [hunk 5]
```diff
@@ -64,7 +66,13 @@ public void tearDown() throws Exception {
 
   @Test
   public void factoryWithDirectUrlWithSpecificBranch() throws Exception {
-    testFactoryWithSpecificBranch.authenticateAndOpen();
+    try {
+
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/factory/DirectUrlFactoryWithSpecificBranch.java]: redundant empty line

**Review by dmytro-ndp — APPROVED** [review]

**Comment by musienko-maxim** [comment]
ci-build

**Pull request merged**